metadata.json

```diff
@@ -44,6 +44,10 @@
     {
       "name": "puppetlabs-powershell",
       "version_requirement": ">= 1.0.1 < 2.0.0"
+    },
+    {
+      "name": "puppetlabs-reboot",
+      "version_requirement": ">= 1.0.1 < 2.0.0"
```

Comment:
Sorry @jpogran - I wasn't necessarily saying you had to remove it... thinking we could ask for a few more opinions on this, as there are some pros and cons to adding reboot as a dependency.

@ferventcoder / @cowofevil any thoughts?